explorer/nodewatch/tests/test_app.py

```diff
@@ -184,12 +184,106 @@ def test_get_api_symbol_nodes_peer(client):  # pylint: disable=redefined-outer-n
 	# Assert: spot check names
 	assert 200 == response.status_code
 	assert 'application/json' == response.headers['Content-Type']
-	assert 5 == len(response_json)
+	assert 7 == len(response_json)
 	assert [
-		'Apple', 'Shin-Kuma-Node', 'ibone74', 'jaguar', 'symbol.ooo maxUnlockedAccounts:100'
+		'Apple', 'Shin-Kuma-Node', 'ibone74', 'jaguar', 'symbol.ooo maxUnlockedAccounts:100', 'xym pool', 'yasmine farm'
 	] == list(map(lambda descriptor: descriptor['name'], response_json))
 
 
+def test_get_api_symbol_nodes(client):  # pylint: disable=redefined-outer-name
+	# Act:
+	response = client.get('/api/symbol/nodes')
+	response_json = json.loads(response.data)
+
+	# Assert: spot check names
+	assert 200 == response.status_code
+	assert 'application/json' == response.headers['Content-Type']
+	assert 8 == len(response_json)
+	assert [
+		'Allnodes250', 'Apple', 'Shin-Kuma-Node', 'ibone74', 'jaguar', 'symbol.ooo maxUnlockedAccounts:100', 'xym pool', 'yasmine farm'
+	] == list(map(lambda descriptor: descriptor['name'], response_json))
+
+
+def test_get_api_symbol_nodes_order_random(client):  # pylint: disable=redefined-outer-name
+	# Act:
+	response = client.get('/api/symbol/nodes?order=random')
+	response_json = json.loads(response.data)
+
+	# Assert: spot check names
+	assert 200 == response.status_code
+	assert 'application/json' == response.headers['Content-Type']
+	assert 8 == len(response_json)
+	expected_names = [
+		'Allnodes250', 'Apple', 'Shin-Kuma-Node', 'ibone74', 'jaguar', 'symbol.ooo maxUnlockedAccounts:100', 'xym pool', 'yasmine farm'
+	]
+	actual_names = list(map(lambda descriptor: descriptor['name'], response_json))
+	assert sorted(expected_names) == sorted(actual_names)
```

Comment:
you want to additionally check the nodes returned are not already sorted. 

but this is probably a better test combined with a limit.